docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,137 @@
+# Kubernetes Deployment
+
+## AnyCable-Go
+
+anycable-go can be easily deployed to your Kubernetes cluster using Helm and [our official Helm chart][anycable-helm].
+
+ 1. Add it as a dependency to your main application:
+
+    ```yaml
+    # Chart.yaml
+    dependencies:
+    - name: anycable-go
+      version: 0.2.4
+      repository: https://helm.anycable.io/
+    ```
+
+    Check latest Helm chart version at [github.com/anycable/anycable-helm/releases](https://github.com/anycable/anycable-helm/releases).
+
+ 1. And then configure it in your application values within `anycable-go` namespace:
+
+    ```yaml
+    # values.yaml
+    anycable-go:
+      env:
+        # Assuming that Ruby RPC is available in K8s in the same namespace as anycable-rpc service (see next chapter)
+        anycableRpcHost: anycable-rpc:50051
+      ingress:
+        enable: true
+        path: /cable
+
+    # values/production.yaml
+    anycable-go:
+      env:
+        # Assuming that Redis is available in K8s in the same namespace as redis-anycable service
+        anycableRedisUrl: redis://:password@redis-anycable:6379/0
+      ingress:
+        acme: # if you're using Let's Encrypt
+          hosts:
+            - your-app.com
+    ```
+
+Read the [chart’s README][anycable-helm] for more info.
+
+## Ruby RPC server
+
+To run Ruby counterpart of Anycable which will handle connection authentication and execute your business logic we will need to create separate deployment and service for it.
+
+ 1. [**Deployment**](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) that will spin up required number of pods and handle rolling restarts on deploys
+
+    ```yaml
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: anycable-rpc
+      labels:
+        component: anycable-rpc
+    spec:
+      replicas: 1
+      strategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: 1
+          maxSurge: 0
+      selector:
+        matchLabels:
+          component: anycable-rpc
+      template:
+        metadata:
+          labels:
+            component: anycable-rpc
+        spec:
+          containers:
+            - name: anycable-rpc
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: IfNotPresent
+              command:
+                - bundle
+                - exec
+                - anycable
+                - !!str --rpc-host=0.0.0.0:50051
+              env:
```

Comment:
Should we mention to include the rest of the env requred by the application itself?

Reply:
Yes 👍